learn/src/main/res/layout/incorrect_output_layout.xml

```diff
@@ -16,19 +16,34 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintStart_toStartOf="parent">
 
-        <com.google.android.material.button.MaterialButton
-            android:id="@+id/btnSeeExplanation"
-            style="@style/Widget.MaterialComponents.Button.TextButton"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:text="@string/see_explanation"
-            android:textStyle="bold"
-            android:textSize="14sp"
-            android:textColor="#2E2E2E"
-            android:textAllCaps="false"
+        <!--        <com.google.android.material.button.MaterialButton-->
+        <!--            android:id="@+id/btnSeeExplanation"-->
+        <!--            style="@style/Widget.MaterialComponents.Button.TextButton"-->
+        <!--            android:layout_width="0dp"-->
+        <!--            android:layout_height="wrap_content"-->
+        <!--            android:text="@string/see_explanation"-->
+        <!--            android:textStyle="bold"-->
+        <!--            android:textSize="14sp"-->
+        <!--            android:textColor="#2E2E2E"-->
+        <!--            android:textAllCaps="false"-->
+        <!--            app:layout_constraintTop_toTopOf="parent"-->
+        <!--            app:layout_constraintStart_toStartOf="parent"-->
+        <!--            app:layout_constraintEnd_toStartOf="@id/btnRetry"/>-->
+
+        <TextView
+            android:id="@+id/miss_txt"
```

Comment:
try to use same convention style as everywhere else. In other places in this file we are using cameCase so use that instead of snake_case